### ISSUE TITLE
WCAG 1.4.3: Disabled Sliders get flagged in Accessibility Insights for Web

### ISSUE THREAD

**Issue by Sentri** [microsoft/fluentui#18223]
### Environment Information

- **Package version(s)**: 7.164.0 (and newer)
- **Browser and OS versions**: Microsoft Edge Version 90.0.818.62 (Official build) (64-bit), Microsoft Windows 10 21H1 19043.985

### Describe the issue:

Accessibility Insights for Web flags a disabled slider's value field for failed color contrast:
![image](https://user-images.githubusercontent.com/753072/118554831-ceb40580-b716-11eb-9874-3dc96226d3c3.png)

WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.ms-Slider-value)
Tags: Accessibility, WCAG 1.4.3, color-contrast
Element path: .ms-Slider-value
Snippet: `<label class="ms-Label ms-Slider-value valueLabel-65">300</label>`

### Please provide a reproduction of the issue in a codepen:
Copied disabled slider example from the documentation, gets flagged by Accessibility Insights FastPass:
https://codepen.io/sentri/pen/ZEepKxx

#### Actual behavior:

Accessibility insights for Web reports a failure on the slider component.

#### Expected behavior:

Accessibility insights for Web should not report issues with a disabled component.

### Documentation describing expected behavior

This is probably caused by the label missing the `for` attribute. The label should point to the slider element's `id` with the `for` attribute, so that Accessible tools can identify the `aria-disabled="true"` attribute in the slider and ignore the disabled label.
